testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -2332,6 +2332,173 @@ const formatTestValue = value => Array.isArray(value) ? "[]" : value;
     });
 });
 
+[false, true].forEach(useItemOption => {
+    QUnit.module(`Public API: Tab options are changed via ${useItemOption ? "itemOption" : "option"} method`, () => {
+        class FormTestWrapper {
+            constructor(useItemOption, onInitializedHandler) {
+                this._useItemOption = useItemOption;
+                this._form = $("#form").dxForm({
+                    onInitialized: onInitializedHandler,
+                    items: [{
+                        itemType: "tabbed",
```

Comment:
Whether or not we want to test this scenario for the 'repaintChangesOnly: true/false' of tabPanelOptions? See T746262.
And, for the 'deferRendering: true/false'? See T819246, T746262, T738898, T744403, T737975, T712311